cookbooks/finite_strain/doc/finite_strain.md

```diff
@@ -68,13 +74,13 @@ selected as material model. As it is derived from the "simple"
 material model, all input parameters for the material properties are read in
 from the subsection `Simple model`.
 
-``` prmfile
+```{literalinclude} finite_strain.part.prm
 ```
 
 ```{figure-md} fig:finite_strain
 <img src="finite_strain.svg" style="width:75.0%" />
 
-Accumulated finite strain in an example convection model, as described in Section <a href="#sec:finite-strain" data-reference-type="ref" data-reference="sec:finite-strain">0.0.1</a> at a time of 67.6&#xA0;Ma. Top panel: Temperature distribution. Bottom panel: Natural strain distribution. Additional black crosses are the scaled eigenvectors of the stretching tensor <span class="math inline"><strong>L</strong></span>, showing the direction of stretching and compression.
+Accumulated finite strain in an example convection model, as described in {ref}`sec:cookbooks:finite_strain` at a time of 67.6&#xA0;Ma. Top panel: Temperature distribution. Bottom panel: Natural strain distribution. Additional black crosses are the scaled eigenvectors of the stretching tensor <span class="math inline"><strong>L</strong></span>, showing the direction of stretching and compression.
```

Comment:
there is something off about the time number ("67.6&#xA0;Ma"). Some leftover html code?